### PR TITLE
created project BookMaster

### PR DESCRIPTION
**Pull request by anushka1403**

### CODE REVIEW

File: src/test/java/com/targetReady/BookMaster/BookMasterApplicationTests.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.targetReady.BookMaster;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class BookMasterApplicationTests {
+
+	@Test
+	void contextLoads() {
```
Comment (akarsh7791) [inline src/test/java/com/targetReady/BookMaster/BookMasterApplicationTests.java:10]: Please auto format all the files IntelliJ -> Code -> Reformat Code.

File: src/test/java/com/targetReady/BookMaster/BookMasterApplicationTests.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.targetReady.BookMaster;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class BookMasterApplicationTests {
```
Comment (akarsh7791) [inline src/test/java/com/targetReady/BookMaster/BookMasterApplicationTests.java:7]:
```suggestion
class LibrarySystemTests {
```

File: src/main/java/com/targetReady/BookMaster/BookMasterApplication.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.targetReady.BookMaster;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class BookMasterApplication {
```
Comment (akarsh7791) [inline src/main/java/com/targetReady/BookMaster/BookMasterApplication.java:7]:
```suggestion
public class LibrarySystemApplication {
```

File: src/main/java/com/targetReady/BookMaster/BookMasterApplication.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
+package com.targetReady.BookMaster;
```
Comment (akarsh7791) [inline src/main/java/com/targetReady/BookMaster/BookMasterApplication.java:1]: com.target.ready.library.system

Comment (anushka1403) [reply src/main/java/com/targetReady/BookMaster/BookMasterApplication.java]: package name changed

File: src/test/java/com/targetReady/BookMaster/BookMasterApplicationTests.java (hunk 1 above)
Comment (anushka1403) [reply src/test/java/com/targetReady/BookMaster/BookMasterApplicationTests.java]: done with all the changes

**Review by akarsh7791 — APPROVED** [review]

**Comment by Shubhani25** [comment]
Reviewed the changes
Looks okay

**Review by Shubhani25 — APPROVED** [review]
Looks fine

**Review by ananya-anni — APPROVED** [review]
Looks fine

**Pull request merged**